README.md

```diff
@@ -1 +1,53 @@
-Observe Kubernetes events, collect metrics & trigger endpoint receivers
\ No newline at end of file
+# ![logo](kubeobserver_logo.png)
+
+Observe Kubernetes events, collect metrics & trigger endpoint receivers
+
+## Quick Start
+
+#### Build from Scratch
+
+```bash
+$ make build
+$ ./kubeobserver
+```
+
+## Configuration
+
+### Kubeobserver configuration
+
+Kubeobserver is configure throw environment variables.
+
+| Variable name | Mandatory | Description | Default |
+| --- | --- | --- | --- |
+| K8S_CLUSTER_NAME | true | the cluster name kubeobserver deployed to (for example: "dev-clsuster") | - |
+| EXCLUDE_POD_NAME_PATTERNS | false | comma separated string. pod watcher will ignore pods that has one or more of this values inside the pod name (for example, when EXCLUDE_POD_NAME_PATTERNS="runner" pod name "ruuner-353332dsdsa" will be ignored) | empty-string |
```

Comment:
consider "A Comma separated string of values to be ignored by the podWatcher. Any pod that has one of these values in its name will be ignored (for example, when EXCLUDE_POD_NAME_PATTERNS="runner" pod name "ruuner-353332dsdsa" will be ignored)"?